pwnlib/elf/elf.py

```diff
@@ -443,6 +443,23 @@ def debug(self, argv=[], *a, **kw):
         import pwnlib.gdb
         return pwnlib.gdb.debug([self.path] + argv, *a, **kw)
 
+    def describe(self):
+        """Prints the several information related with the binary protection
+        measures as well as the architecture.
+
+        Examples:
+            >>> e = ELF("/bin/ls", checksec=False)
+            >>> e.describe() #doctest: +SKIP
```

Comment:
Please do not skip doctests unless absolutely necessary.  It's much preferred to use ELLIPSIS than SKIP.

If you intend for an example to *EXCLUSIVELY* be an example (and not a doctest), please use:

```
.. code-block:: python

    >>> Your code goes here
```

Reply:
Still needs to be fixed

Reply:
Done in 13d3555  (put example as python block)